mesonbuild/interpreter.py

```diff
@@ -1877,6 +1877,8 @@ def add_section(self, section, values, kwargs, subproject):
                 elif isinstance(i, (ExternalProgram, Dependency)):
                     FeatureNew.single_use('dependency or external program in summary', '0.57.0', subproject)
                     formatted_values.append(i.summary_value())
+                elif isinstance(i, coredata.UserOption):
+                   formatted_values.append(i.printable_value()) 
```

Comment:
It's missing featurenew just like a couple lines above, to mark it new in 0.58.0

Reply:
What do you think reads better:

- `feature in summary`
- `UserOption` in summary

Reply:
UserOption is an internal object name, not something we should print to the user. Also this is not limited to feature options, or is it? What happens for other option types?